### DISCUSSION TITLE
assets.go:5:12: pattern backend-assets/*: no matching files found

### DISCUSSION THREAD

**Discussion by yurivict** [mudler/LocalAI#1409] [Q&A]
**LocalAI version:**
1.40.0

**Environment, CPU architecture, OS, and Version:**
FreeBSD 13.2
go-1.21

**Describe the bug**
```
assets.go:5:12: pattern backend-assets/*: no matching files found
*** Error code 1
```

**Comment by mudler** [comment]
looks like you failed in compiling backend-assets. Did you used the container images?

**Reply by BennisonDevadoss** [reply]
I attempted to utilize the Docker container images, but encountering issues, I switched to running it without Docker, following the approach of the contribution setup. While setting up the contribution setup I got this exception

**Comment by yurivict** [comment]
> Did you used the container images?

The sources are downloaded from the GoLang server.

**Comment by yurivict** [comment]
@mudler 

Build just fails for me:
```
$ git clone --recurse-submodules https://github.com/mudler/LocalAI.git
...
$ cd LocalAI
$ go121 build
assets.go:5:12: pattern backend-assets/*: no matching files found
```

Do you know what might be wrong?

FreeBSD 14.0
